newIDE/app/src/UI/Theme/DefaultDarkTheme/theme.json

```diff
@@ -156,6 +156,44 @@
         }
       }
     },
+    "search": {
```

Comment:
Could you add those new fields in `newIDE/app/scripts/theme-templates/theme.json` ?

Reply:
Speaking of which, should this field be `searchBar` to be more accurate?